### ISSUE TITLE
When you use the NeumorphShapeDrawable in a RecyclerView item, scroll becomes laggy

### ISSUE THREAD

**Issue by EvGem7** [fornewid/neumorphism#25]
RecyclerView is configured correctly: only a few ViewHolders are created, therefore views shouldn't be recreated.
My item view consists of TextView within NeumorphCardView.
```
<?xml version="1.0" encoding="utf-8"?>
<soup.neumorphism.NeumorphCardView xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:layout_marginStart="4dp"
    android:layout_marginTop="-3.1dp"
    android:layout_marginEnd="4dp"
    android:layout_marginBottom="-3.1dp"
    app:neumorph_shadowColorDark="?attr/neumorphDarkColor"
    app:neumorph_shadowColorLight="?attr/neumorphLightColor"
    app:neumorph_shadowElevation="4dp"
    app:neumorph_shapeAppearance="@style/NeumorphRounded5dp"
    app:neumorph_shapeType="flat">

    <TextView
        android:id="@+id/itemGroupText"
        style="@style/Button.Cancel"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:gravity="start"
        android:padding="16dp"
        android:textSize="16sp"
        tools:text="Group 1" />

</soup.neumorphism.NeumorphCardView>
```

Maybe issue is due to `dirty: Boolean` variable in NeumorphShapeDrawable. It could become true for some reason when it is used in RecyclerView. I think so because profiler says that Shape.updateShadowBitmap() takes a lot of time.
![image](https://user-images.githubusercontent.com/33933105/86118617-bc6ab200-bad9-11ea-839c-ff2a51b3b505.png)

**Comment by fornewid** [comment]
First, thanks for reporting.
I think, it can cause problems when used with `wrap_content` in RecyclerView.
```xml
<soup.neumorphism.NeumorphCardView
    android:layout_height="wrap_content"
    />
```

The root cause is that this library uses a bitmap to implement neumorphism background.
But I haven't come up with any concrete alternatives yet. 🤔 
Please share any good ideas if you have.
Of course, if I also have a good idea, I will implement it.

**Comment by EvGem7** [comment]
@fornewid yeah I've seen that you use a bitmap for background. But item views in RecyclerView are not recreated so I expected bitmaps aren't recreated too. I can spend time on the weekend to investigate why wrap_content causes this. If I find a solution I will create a pull request.

**Comment by paradox02** [comment]
This is still problem, please do you have any ideas for fix it?
thanks

**Comment by fornewid** [comment]
Fixed in `0.3.0`.
